tests/symbolicator/test_sourcemap.py

```diff
@@ -3,35 +3,31 @@
 import pytest
 
 from sentry.models import File, Release, ReleaseFile
-from sentry.testutils import RelayStoreHelper, TransactionTestCase
+from sentry.testutils import RelayStoreHelper
 from sentry.testutils.helpers.datetime import before_now, iso_format
 from tests.symbolicator import insta_snapshot_javascript_stacktrace_data
 
 # IMPORTANT:
 # For these tests to run, write `symbolicator.enabled: true` into your
 # `~/.sentry/config.yml` and run `sentry devservices up`
 
-# NOTE:
-# Run with `pytest --reuse-db tests/symbolicator/test_sourcemap.py -s` until DB issues are resolved.
-
 
 def get_fixture_path(name):
     return os.path.join(os.path.dirname(__file__), "fixtures", name)
 
 
-class SymbolicatorSourceMapIntegrationTest(RelayStoreHelper, TransactionTestCase):
```

Comment:
`TransactionTestCase` is a child class of `BaseTestCase` which wires up testing. If we don't use it (and we cant with parametrization), we need to use the `TestSomething` naming convention for tests to be picked up.